recipes-multimedia/gstreamer/gstreamer1.0-plugins-good_1.16.%.bbappend

```diff
@@ -10,8 +10,14 @@ SRC_URI_append = " \
     file://0007-qtdemux-Allow-streams-with-no-specified-protection-s.patch \
     file://0008-qtdemux-Parse-VP-Codec-Configuration-Box.patch \
     file://0009-qtdemux-added-support-for-cbcs-encryption-scheme.patch \
+    file://0010-qtdemux-No-need-for-new-application-x-cbcs-caps.patch \
 "
-
+APPLY_WPE_2.28_BUILD ??= "false"
```

Comment:
Please rename the variable. It have be something with WPEWEBKIT. Perhaps it's possible to read webkit package version instead (PREFERRED_VERSION_wpewebkit?) and don't depend on additional value to set

Reply:
sure will do

Reply:
done